pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1502,4 +1502,8 @@ self: super: {
     excludes = [ "stack.yaml" "sources.json" "src/Cachix/Types/Session.hs" "src/Cachix/API/Signing.hs" "cachix-api.cabal" "workflows/test.yml" ];
   });
 
+  # 2020-11-23: Disable tests due to an incorrect call of `fail` on a Monad
+  # rather than a MonadFail
+  blas-hs = dontCheck super.blas-hs;
```

Comment:
Could you report this upstream, and link it here in the code so we know when it can be removed?

Reply:
Done.